.sauce/config.yml

```diff
@@ -1,8 +1,10 @@
 apiVersion: v1alpha
 kind: cypress
+defaults:
+  mode: sauce
 sauce:
   region: us-west-1
-  concurrency: 2 # Controls how many suites are executed at the same time.
+  concurrency: 1 # Controls how many suites are executed at the same time.
```

Comment:
I'd leave this as more than one :)

Reply:
oh make sense! Let me update this. and also the demo gif.